packages/gatsby-theme-docs/src/components/user-research-banner.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import css from '@emotion/css';
+import { designSystem } from '@commercetools-docs/ui-kit';
+import ctLogo from '../../static/ct-logo.png';
+import Link from './link';
+
+const Banner = styled.div`
+  div {
+    max-width: ${designSystem.dimensions.widths.userResearchBannerMaxWidth};
+    border-top: 1px solid ${designSystem.colors.light.borderHighlight};
+  }
+  img {
+    margin: 0 0 -5px -5px;
+  }
+  p {
+    font-size: ${designSystem.typography.fontSizes.small};
+  }
+`;
+
+const UserResearchBanner = () => {
+  return (
+    <Banner>
+      <div>
+        <img width="22px" height="22px" src={ctLogo}></img>
```

Comment:
Not sure where to get a svg or an image of the ct-logo without space around it 🤔

Reply:
I think we should have the CT logos made available by the docs ui kit as components.  Concerning the spacing around it you have to ask Sven, I remember someone telling me that the spacing is required and sven's design does actually have more spacing as opposed to this implementation (at least to the top and bottom, the left spacing could have to be overridden in CSS.

Reply:
You should use the one from appkit `@commercetools-frontend/assets` package, which is an SVG.

https://github.com/commercetools/merchant-center-application-kit/blob/636fc45c0d53544a90b0977b2212ec3a8aec8b94/packages/application-shell/src/components/app-bar/app-bar.tsx#L9

Reply:
Ah, thanks!! 👍